contracts/process/pipeline.go

```diff
@@ -35,10 +43,10 @@ type Pipeline interface {
 	OnOutput(handler OnPipeOutputFunc) Pipeline
 
 	// Run builds, executes, waits for completion, and returns the final Result.
-	Run(func(builder Pipe)) (Result, error)
+	Run(configurer ...func(Pipe)) (Result, error)
 
 	// Start builds and starts execution asynchronously, returning a RunningPipe.
-	Start(func(builder Pipe)) (RunningPipe, error)
+	Start(configurer ...func(Pipe)) (RunningPipe, error)
```

Comment:
Can the variable be empty? It may be unnecessary for now.


```suggestion
	Run() (Result, error)

// Start builds and starts execution asynchronously, returning a RunningPipe.
	Start() (RunningPipe, error)
```